juju/client/facade.py

```diff
@@ -350,6 +350,8 @@ def buildTypes(schema, capture):
     for kind in sorted((k for k in schema.types if not isinstance(k, str)),
                        key=lambda x: str(x)):
         name = schema.types[kind]
+        if not name:
```

Comment:
A comment would be useful here.

Reply:
Yeah you got me thinking and I might try to further debug this weird case where an empty name comes up in the  3.1.0 client-only codegen (not with any other client-only schemas, and not with 3.1.0 full schema).